consensus/polybft/blockchain_wrapper.go

```diff
@@ -224,7 +224,12 @@ func (p *blockchainWrapper) UnubscribeEvents(subscription blockchain.Subscriptio
 }
 
 func (p *blockchainWrapper) GetChainID() uint64 {
-	return uint64(p.blockchain.Config().ChainID)
+	chainID := p.blockchain.Config().ChainID
+	if chainID < 0 {
```

Comment:
better exclude the rule - chainId is always > 0

Reply:
Then, I will have to use the `//no-lint` tag. Is that okay? The linter warns us that there is a possibility of error in the conversion.